### DISCUSSION TITLE
aria2下载完成后不能转移到指定目录，报错unaligned 64-bit atomic operation

### DISCUSSION THREAD

**Discussion by shcat10** [AlistGo/alist#1840] [Q&A]
环境：win service 2019
日志：
[31mERRO[0m[2022-09-28 12:02:07] error [unaligned 64-bit atomic operation] while run task [download http://www.newbook8.com/ISBN9787544070775.pdf to [/F盘](F:\)] 
![image](https://user-images.githubusercontent.com/44720659/192685433-a7a20f26-dfdf-4cfa-a2c3-d44180ab098e.png)

**Comment by 4ppp44** [comment]
我也遇到了
